modules/renovate-bot/renovate.tf

```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "renovate" {
+  name      = "renovate"
+  namespace = "renovate"
```

Comment:
can we have helm release name and namespace as config variable, to allow to override when needed